templates/components/cart/content.html

```diff
@@ -122,17 +122,15 @@ <h4 class="cart-item-name"><a href="{{url}}">{{name}}</a></h4>
                     {{else}}
                         {{> components/common/login-for-pricing}}
                     {{/or}}
-                    {{#if can_modify}}
-                        <a class="cart-remove icon" data-cart-itemid="{{id}}" href="#" data-confirm-delete="{{lang 'cart.confirm_delete'}}">
+                    {{#or can_modify (if type '==' 'GiftCertificate')}}
+                        <button class="cart-remove icon"
+                                data-cart-itemid="{{id}}"
+                                data-confirm-delete="{{lang 'cart.confirm_delete'}}"
+                                aria-label="{{lang 'cart.remove_item'}}"
+                        >
                             <svg><use xlink:href="#icon-close"></use></svg>
```

Comment:
this icon is decorative, maybe we should add aria-hidden="true" to hide it from screen readers?

Reply:
It's not readable by VoiceOver and ANDI now